x-pack/plugins/ml/public/application/datavisualizer/datavisualizer_selector.tsx

```diff
@@ -50,8 +50,11 @@ function startTrialDescription() {
 
 export const DatavisualizerSelector: FC = () => {
   useTimefilter({ timeRangeSelector: false, autoRefreshSelector: false });
+  const {
+    services: { licenseManagement },
+  } = useMlKibana();
 
-  const startTrialVisible = isFullLicense() === false;
+  const startTrialVisible = licenseManagement.enabled === true && isFullLicense() === false;
```

Comment:
As the licenseManagement plugin is optional, you first need to check for the plugin existence.

Reply:
Changes in 73e832b65c118e802e2e5bb1a85d834bee95f5af